pkg/loadbalancer/config.go

```diff
@@ -102,10 +108,15 @@ type LBMapsConfig struct {
 	MaglevMapMaxEntries                                             int
 }
 
+const defaultMaxSockRevNatEntries = 256 * 1024
```

Comment:
Why this number in particular?

Reply:
Unclear to me why this was chosen originally (https://github.com/cilium/cilium/blob/main/pkg/maps/lbmap/ipv4.go#L29)